src/utils/tail.js

```diff
@@ -45,4 +45,4 @@ const tail = filePath => {
   }
 }
 
-module.exports = tail
+module.exports = Tail
```

Comment:
The name of the function is irrelevant as it gets exported directly. You could just do
```javascript
module.exports = filePath => {
  ...
}
```

Reply:
thanks - changed in https://github.com/rowolff/babel-poe/commit/8d18bd30cf24aa5442c1e14cf46ec446c10b41b3